server/src/test/java/org/cloudfoundry/identity/uaa/zone/ZoneAwareClientSecretPolicyValidatorTests.java

```diff
@@ -39,6 +39,8 @@ void setUp() {
     @Test
     void testEmptyClientSecret() {
         zone.getConfig().setClientSecretPolicy(defaultPolicy);
+        validator.validate(TEST_SECRET_1);
```

Comment:
It's also not obvious to understand that this line is actually testing something. It really looks like some sort of test setup. People could miss that we are actually expecting this call not to throw any exception. 

The typical way of doing what this test is doing is to create 2 different tests. One that tests with the default policy, and one that tests with the strict policy.

Reply:
split tests and renamed variable